### DISCUSSION TITLE
How to grease Pencil in 4.3+?

### DISCUSSION THREAD

**Discussion by gerroon** [nortikin/sverchok#5169] [Q&A]
Hi

How do we modify existing grease pencil drawings in 4.3? The OBJ ID selector does not let me pick the grease pencil object, not even listing it. I want to read and modify GP drawings.

![image](https://github.com/user-attachments/assets/387e2ecf-81b9-47c0-ae78-e3c8e9f0d8a5)

**Comment by nortikin** [comment]
start GP in scene and it will appear
